build.gradle

```diff
@@ -11,15 +11,23 @@ buildscript {
     }
 }
 
+def sonatypeRepo = 'https://oss.sonatype.org/service/local/staging/deploy/maven2/'
+def sonatypeSnapshots = 'https://oss.sonatype.org/content/repositories/snapshots/'
+
+def distributionRepository = properties["distributionRepository"] ?: sonatypeRepo
+def snapshotsRepository = properties["snapshotsRepository"] ?: sonatypeSnapshots
```

Comment:
I included a small unrelated change which allows parameterization of the deploy repositories. This will allow me to easily test integration of this JPMS support with downstream libraries. It should be completely inert to this codebase.